flink-table/flink-table-planner-blink/src/test/resources/org/apache/flink/table/planner/plan/rules/logical/PushFilterIntoTableSourceScanRuleTest.xml

```diff
@@ -51,38 +70,45 @@ LogicalProject(name=[$0], id=[$1], amount=[$2], virtualField=[$3], price=[$4])
       <![CDATA[
 LogicalProject(name=[$0], id=[$1], amount=[$2], virtualField=[$3], price=[$4])
 +- LogicalProject(name=[$0], id=[$1], amount=[$2], virtualField=[+($2, 1)], price=[$3])
-   +- LogicalFilter(condition=[>($3, 10)])
+   +- LogicalFilter(condition=[OR(>($2, 2), <($2, 10))])
       +- LogicalTableScan(table=[[default_catalog, default_database, VirtualTable, filter=[]]])
 ]]>
     </Resource>
   </TestCase>
-  <TestCase name="testCannotPushDown3">
+  <TestCase name="testWithNullLiteral">
```

Comment:
If you add this to the end of the file, it won't show so much diffs in this PR, which would help the reviewer a lot.